docs/guide/units-coordinates.rst

```diff
@@ -165,7 +165,7 @@ frames defined in Astropy, for example `~astropy.coordimates.ICRS`.
 
   >>> c.transform_to('icrs')
   <SkyCoord (ICRS): (ra, dec, distance) in (deg, deg, km)
-    (49.85053118, 0.05723938, 1417577.02975451)>
+    (49.85053118, 0.05723938, 1417577.0297545...)>
```

Comment:
I HEAR YOU LIKE A HACK|

Reply:
What happened here is that the 1 disappeared in the 32bit versions but was present in 64 bit builds. 
This way, we just ignore extra decimal place but it looks ugly.